.automatedtesting.json

```diff
@@ -12,10 +12,5 @@
     "ENABLE_GEMS": [
     ],
     "REPOS": [
-        {
-            "NAME": "o3de-extras",
-            "URL": "https://github.com/o3de/o3de-extras.git",
-            "BRANCH": "development"
-        }
```

Comment:
Calling @amzn-changml 
This change should not be merged over. 
It is part of the .gitignore in the stabilization/2310 repo and it's purpose is to stop o3de-extras from being built in the stabilization branch

Reply:
Done, anything else?

Reply:
No, this should be good.

Reply:
Yup, thanks!